httpobs/scanner/analyzer/headers.py

```diff
@@ -110,15 +114,24 @@ def content_security_policy(reqs: dict, expectation='csp-implemented-with-no-uns
         output['result'] = 'csp-header-invalid'
         return output
 
-    # If we have headers but they're blank, that should error out
-    if (any('Content-Security-Policy' in h for h in [response.headers, response.http_equiv])
-       and not any(headers.values())):
-        output['result'] = 'csp-header-invalid'
-        return output
-    elif not any(headers.values()):
+    # If we have neither HTTP header nor meta, then there isn't any CSP
+    if not any(headers.values()):
         output['result'] = 'csp-not-implemented'
```

Comment:
I'd like to see these magic values stored as constants, but that's preference/nitty.

Reply:
There shouldn't be any magic values here. It's basically this:

```python
headers = {
  'http': None,  # no CSP header via CSP
  'meta': None   # no CSP header via http-equiv
}
```

It's basically just checking to see if they're both None. Should I write it as:
`if headers['http'] is None and headers['meta'] is None:`

That would probably be clearer.

Reply:
By "magic value" I meant `'csp-not-implemented'`, which you may want to be able to compare the values of in tests.

Reply:
Ahh! Each test returns "magic values" in the sense that it returns a text-based result code like that. It's not used outside the function so I think it's pretty safe in this case to not make it a Magic value.